src/utils/LazyLoader/plotly.js

```diff
@@ -0,0 +1,6 @@
+export default () => Promise.resolve(window.Plotly ||
```

Comment:
While we're here, might be interesting to check if `window.Plotly.version` matches our expected minimal version, and if not, force loading the one in our bundle.. this will also be useful to protect against old versions of dash-bio dumping an old version of Plotly.js for DCC to use.